torch_geometric/nn/aggr/multi.py

```diff
@@ -114,6 +116,14 @@ def reset_parameters(self):
         if hasattr(self, 'multihead_attn'):
             self.multihead_attn._reset_parameters()
 
+    def get_out_channels(self, in_channels):
+        if self.out_channels is not None:
+            return self.out_channels
+        if self.mode == 'cat':
```

Comment:
This may not work. If each aggregator has customized `out_channels` such as `LSTMAggregation`. Do we want to add `get_out_channels` to all the `Aggregation`?

Reply:
We can maybe add it as a TODO? I think having support for standard aggregators such as mean, max and mean is sufficient for now.

Reply:
Sounds good